src/StatementDumper.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Helmich\TypoScriptParser;
```

Comment:
I'm wondering if this might also fit in the `Parser` namespace; that's where all the other classes that operate on the AST also live (like the printer or the traverser implementations).